USF-2012: Create Terms and Conditions documentation (#204)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchant/terms-and-conditions.mdx

---

> title: Terms & Conditions setup

Reviewer: ```suggestion
title: Set up Terms & Conditions
```

> description: Learn how to configure, manage and display the Terms & Conditions feature during checkout process.

Reviewer: ```suggestion
description: Learn how to enable and set up the Terms & Conditions feature.
```

---

import Aside from '@components/Aside.astro';
import Callouts from '@components/Callouts.astro';
import { Steps } from '@astrojs/starlight/components';
import Tasks from '@components/Tasks.astro';
import Task from '@components/Task.astro';

> The Terms & Conditions feature must be enabled and configured in order to be used during a guest checkout process before placing the order. The terms and conditions configured will be displayed on the Checkout component inside the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) as checkboxes within a label containing text and links directing them to the appropriate configured agreements pages.

Reviewer: ```suggestion
The Terms & Conditions feature must be enabled and configured to be available during the guest checkout process before placing an order.

The configured terms and conditions will appear in the Checkout component within the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) as specific checkboxes. Each checkbox will display a label with text and links, directing users to the corresponding agreement pages.
```

<Aside type="tip" title="TermsAndConditions configurable on a per store-view basis">

>   Merchants can add as many agreements as they want for a specific store-view; this allows them to provide different documents for each language and other needs.

Reviewer: ```suggestion
  Merchants can add as many agreements as they want for a specific store-view.
```

>   Please visit [Multi-store setup](/merchant/multistore/) to know exactly how to build blocks and configure the storefronts structure based on **Edge Delivery Services (EDS)**.

Reviewer: ```suggestion
  Visit the [Multi-store setup](/merchant/multistore/) documentation for more information on how to build blocks and configure the storefront structure based on **Edge Delivery Services (EDS)**.
```

</Aside>

## How to enable and configure the Terms and Conditions

This checkout component is configurable on a per store-view basis on EDS, but a merchant can also rely on the Admin Panel's configuration.

> Please follow the next step-by-step guide to enable and configure the Terms and Conditions:

Reviewer: ```suggestion
Follow the next step-by-step guide to enable and configure the Terms and Conditions:
```

<Tasks>

<Task>

### Enable the component

> The merchant has to enable the Terms and Conditions component using the Admin Panel as following:

Reviewer: ```
To enable the component in the Admin Panel:

<Steps>

1. Log in to the Admin Panel.

1. Navigate to **Stores** > **Settings** > **Configuration**.

1. Under **Sales**, select **Checkout**.

1. In **Checkout Options**, set **Enable Terms and Conditions** to Yes.

</Steps>

```

<Steps>

1. Log in to the Admin Panel dashboard.
1. Go to the _"Stores"_ menu and click on the _"SETTINGS" > "Configuration"_ option.
1. Navigate to the _"SALES"_ section and click on the _"Checkout"_ option.
1. Finally, into the _"Checkout Options"_ enable the _"Enable Terms and Conditions"_ configuration selecting the _**Yes**_ value for the store-view desired.

</Steps>

> See [Terms and conditions for checkout](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/point-of-purchase/checkout/terms-and-conditions) documentation to enable the terms and conditions using the Admin Panel for more details.

Reviewer: ```suggestion
See [Terms and conditions for checkout](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/point-of-purchase/checkout/terms-and-conditions) documentation for more information on how to enable the terms and conditions using the Admin Panel.
```

    Similarly, the merchant can configure the translation for the validation error that appears when the user clicks the `Place Order` button and an agreement is not checked, using the **"Checkout.TermsAndConditions.error"** identifier.

    Sample of **placeholders.xlsx** file:
    ```xlsx
    "Checkout.TermsAndConditions.label", "I have read, understand, and accept the <a href='/en/privacy-policy' target='_blank'>Terms of Use, Terms of Sales, Privacy Policy, and Return Policy</a>."
    "Checkout.TermsAndConditions.terms_label", "By placing an order, you are agreeing to be bound by our <a href="/en/terms-and-conditions" target="_blank">Terms And Conditions</a>."
    "Checkout.TermsAndConditions.privacy_label", "By placing an order, you acknowledge our <a href="/en/privacy-policy" target="_blank">Privacy Policy</a>."
    "Checkout.TermsAndConditions.error", "Please accept the Terms and Conditions to continue."
    ```

    These labels will be used in order to render the `TermsAndConditions` container inside of the **commerce-checkout.js** block file.

>     Please visit the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) to see some examples of how these labels will be used.

Reviewer: ```suggestion
    Visit the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) documentation to see examples on how to use these labels.
```

    - **`Applied`**
      Specifies the mode how the checkbox should appear:
      - 'Manually': the user should manually check the terms
      - 'Automatically': the checkbox will appear checked by default
    - **`Store View`**
      Defines the scope at store level for this agreement
    - **`Checkbox Text`**
      It contains directly the text to show; it accepts HTML with links to a specific page in EDS

    <Aside type="note" title="Some attributes not utilized">
      Although the following attributes are provided by the Admin Panel, they are not going to be utilized in the current solution because they will be handled in EDS. Merchants are able to work on their own solution if they want to use them:

      - **`Content`**
        As described above, the content for the agreements pages will be created **ONLY** using EDS
      - **`Show Content as`** (Text, HTML)
        Not really needed because EDS pages accepts HTML code
      - **`Content Height (css)`**
    </Aside>

    These entries will be used in order to render the `TermsAndConditions` container inside of the **commerce-checkout.js** block file.
    

>     Please visit the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) to see some examples of how they will be used.

Reviewer: ```suggestion
    Visit the [TermsAndConditions container](/dropins/checkout/containers/terms-and-conditions/) documentation to see examples of this feature.
```

</Steps>

> See [Terms and conditions for checkout](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/point-of-purchase/checkout/terms-and-conditions) documentation to configure the terms and conditions using the Admin Panel for more details.

Reviewer: ```suggestion
See the [Terms and conditions for checkout](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/point-of-purchase/checkout/terms-and-conditions) documentation to learn how to configure the terms and conditions using the Admin Panel.
```
